pandas/_libs/tslibs/strptime.pyx

```diff
@@ -140,13 +140,13 @@ def array_strptime(object[:] values, object fmt,
                     iresult[i] = NPY_NAT
                     continue
                 raise ValueError("time data %r does not match "
-                                 "format %r (match)" % (values[i], fmt))
+                                 "format %r (match)" % (val, fmt))
```

Comment:
Do you mind converting these to use `format` while already modifying these?

Reply:
I've just tried to do it but a couple of tests failed. I don't know the reason yet.